client/src/components/tabs.js

```diff
@@ -85,16 +85,16 @@ export default class Tabs extends React.Component{
                     This is based on the historical growth data of each country that is currently ahead of {country_name} in the outbreak.
                     </p>
                     <p className="is-size-6">
-                    Viewing this can offer unique insights into the range of possible outcomes. Not based on epidemiological models, only on historical data experienced by other countries.
+                    Viewing this can offer unique insights into the range of possible outcomes. The forecast is not based on epidemiological models, just on historical data experienced by other countries.
                     </p>
                     <p className="is-size-6">
-                        The potential forecast does not take into account the relative doubling time of each country.
+                        The forecast does not take into account the relative doubling time of each country.
                     </p>
                     <p className="is-size-6">
-                        The true forecast depends on a multitude of factors such as the number and speed of tests done, the quality of the case tracking, the testing of tracked cases, and the support given to those who need to go into isolation.
+                        Forecasting accuracy depends on a multitude of factors such as the number and speed of tests done, the quality of the case tracking, the testing of tracked cases, and the support given to those who need to go into isolation.
```

Comment:
Avoiding the notion of a "true forecast" since by the time it's certainly true, it's probably not a forecast.